db/migrate/20220720041810_reshare_table_tuning.rb

```diff
@@ -0,0 +1,6 @@
+class ReshareTableTuning < ActiveRecord::Migration[5.2]
+  def change
+    add_column :reshare_patron_requests, :__start, :datetime
```

Comment:
When we ingest via CSV, any headers like __start will be automatically converted to "start". So having a column that starts with not a letter will cause issues down the line.